datafusion/core/README.md

```diff
@@ -19,8 +19,15 @@
 
 # DataFusion Core
 
-[DataFusion][df] is an extensible query execution framework, written in Rust, that uses Apache Arrow as its in-memory format.
+[DataFusion][df] is an extensible query execution framework, written in Rust,
+that uses Apache Arrow as its in-memory format.
 
-This crate contains the main entrypoints and high level DataFusion APIs such as SessionContext, and DataFrame and ListingTable.
+This crate contains the main entrypoints and high level DataFusion APIs such as
+SessionContext, and DataFrame and ListingTable.
 
-[df]: https://crates.io/crates/datafusion
+For more information:
+
+- [DataFusion Website](https://arrow.apache.org/datafusion)
+- [DataFusion API Docs](https://docs.rs/datafusion/latest/datafusion/)
+
+[df]: https://arrow.apache.org/datafusion
```

Comment:
This is same link as `DataFusion Website` above. I think we can remove this?

Reply:
Good idea, I will do so